pywwt/layers.py

```diff
@@ -285,6 +337,17 @@ def __init__(self, parent=None, table=None, frame=None, **kwargs):
         self.table = table
         self.frame = frame
 
+        # ISSUE: For some reason, WWT seems to prefer the that time column
+        # be 0th-10th/12th column in the table.
+        ### (varies with placement of latitude and longitude columns,
+        ###  which also need to be 0th-12th)
+        # So the current proposition is to limit table's max number of columns
+        ### (the max will change depending on how time_att is implemented)
+        max_cols = 13
```

Comment:
I think rather than do this it would be better to check when ``time_att`` is set whether it is in within the range of columns is 0-12, and give an error if not explaining that this won't work for now. I've definitely tried to plot tables with more than 13 columns before and I don't think we want to constrain things in all the cases where time is not used.

Of course, it'd be even better if we can figure out what is causing this limitation :) Maybe @jsub1 has some ideas?

Reply:
@jsub1 You're right, it's a parsing issue with the `place` column, not the number of columns in the table. I'll take this part out in the next commit.